client/my-sites/drafts/author-selector.jsx

```diff
@@ -0,0 +1,54 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import i18n from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import Gravatar from 'components/gravatar';
+import NavSegmented from 'components/section-nav/segmented';
+import NavItem from 'components/section-nav/item';
+import userLib from 'lib/user';
+const user = userLib();
+
+function AuthorSelector( { selectedScope, onChange } ) {
+	const scopes = [
+		{
+			id: 'me',
+			caption: i18n.translate( 'Me', { context: 'Filter label for posts list' } ),
```

Comment:
Same issue as noted in your other pull request: https://github.com/Automattic/wp-calypso/pull/8539#discussion_r83031773